c2corg_ui/static/js/documentediting.js

```diff
@@ -374,6 +389,25 @@ app.DocumentEditingController.prototype.handleMapFeatureChange_ = function(
 
 
 /**
+ * It doesn't associate, it just pushes it into associations array.
+ * @private
+ */
+app.DocumentEditingController.prototype.pushDocToAssociations_ = function() {
```

Comment:
I made it public in another pr because I needed it a part of other functions. It was only accessed through `readDocument` and `readAccount`. ReadDocument sounds ok, but this function forces me to use a URL that I don't want. 
Is it ok to make getJson public ?